src/handler.ts

```diff
@@ -56,6 +59,15 @@ export default class DataHandler {
    * @param event a browser Event or CustomEvent emitted
    */
   handleEvent(event: CustomEvent<DataLayerDetail>): void {
+    const handleEventSpan = Telemetry.startSpan(telemetryType.handleEventSpan, {
+      operatorCount: this.operators.length,
+      operatorNames: this.operators
```

Comment:
Is this worth collecting? Figured knowing which operators are used and to what degree would be valuable. Is comma separated the best way to go? Should these at least be alphabetically sorted for consistency?

Reply:
I think insights about operators is interesting and including the operator chain in the event-handling performance metric should give us insights into whether there are any unexpected latencies in our rule sets - could be interesting for tuning default rules or understanding performance implications of custom rules.